### ISSUE TITLE
Set sidebar->dimmer to true will make element on page unclickable

### ISSUE THREAD

**Issue by LEAFERx** [theme-next/hexo-theme-next#820]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 

When sidebar is open, I can still click links/buttons and select text on page normally.

### Actual behavior 

When sidebar is open, I can not click links/buttons directly. I have click twice, one for close sidebar one for clicking links/buttons. Also, I can not select text when sidebar is open.

### Steps to reproduce the behavior 
1. in theme _config.yml, set sidebar->dimmer to true
2. visit any blog page
3. open sidebar

* Link to demo site with this bug: any post at https://leaferx.online
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```
v10.4.1
6.1.0
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.8.0"
  },
  "dependencies": {
    "babel-core": "^6.26.3",
    "babel-preset-es2015": "^6.24.1",
    "babelify": "^8.0.0",
    "browserify": "^16.2.3",
    "gulp-babel": "^6.1.3",
    "gulp-htmlclean": "^2.7.20",
    "gulp-htmlmin": "^4.0.0",
    "gulp-minify-css": "^1.2.4",
    "gulp-uglify": "^3.0.0",
    "hexo": "^3.8.0",
    "hexo-autonofollow": "^1.0.1",
    "hexo-baidu-url-submit": "0.0.5",
    "hexo-deployer-git": "^0.3.1",
    "hexo-generator-archive": "^0.1.5",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-feed": "^1.2.2",
    "hexo-generator-index-pin-top": "^0.2.2",
    "hexo-generator-searchdb": "^1.0.8",
    "hexo-generator-sitemap": "^1.2.0",
    "hexo-generator-tag": "^0.2.0",
    "hexo-leancloud-counter": "^0.2.10",
    "hexo-leancloud-counter-security": "^1.3.1",
    "hexo-renderer-ejs": "^0.3.1",
    "hexo-renderer-markdown-it": "^3.4.1",
    "hexo-renderer-stylus": "^0.3.3",
    "hexo-server": "^0.3.3",
    "hexo-symbols-count-time": "^0.3.2",
    "markdown-it-footnote": "^3.0.1"
  },
}

```

#### Hexo version

```
hexo: 3.8.0
hexo-cli: 1.1.0
os: Windows_NT 10.0.17134 win32 x64
http_parser: 2.8.0
node: 10.4.1
v8: 6.7.288.45-node.7
uv: 1.20.3
zlib: 1.2.11
ares: 1.14.0
modules: 64
nghttp2: 1.32.0
napi: 3
openssl: 1.1.0h
icu: 61.1
unicode: 10.0
cldr: 33.0
tz: 2018c
```

#### Hexo Configuration

not relevant

### NexT Information

**NexT Version:**

- [x] Latest Master branch
- [ ] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [x] Muse
- [ ] Mist
- [ ] Pisces
- [ ] Gemini

#### NexT Configuration:

```yml
sidebar:
  dimmer: true
```

### Other Information 
N/A

**Comment by stevenjoezhang** [comment]
You can set `sidebar->dimmer` to `false`

**Comment by 1v9** [comment]
https://github.com/theme-next/hexo-theme-next/blob/master/source/css/_common/components/sidebar/sidebar-dimmer.styl

**Comment by LEAFERx** [comment]
Solutions maybe:
1. add event listener to body, when click on non-interactive element (eg. not links/buttions/text/not-binded-click-event) and outside of sidebar, close sidebar.

or

2. add a option that dimmer-only-on-mobile, cause it seems not very useful on big screen device but on small screen device sidebar hide the post and there's shadow outside the sidebar.

**Comment by stevenjoezhang** [comment]
https://github.com/theme-next/hexo-theme-next/issues/606
https://github.com/theme-next/hexo-theme-next/pull/607

**Comment by stevenjoezhang** [comment]
Fixed in #964
